src/index.html

```diff
@@ -20,8 +21,437 @@
       href="styles/index.scss"
```

Comment:
Link to the SCSS file should be a CSS file in the HTML. Browsers do not understand SCSS syntax, so ensure you are linking to a compiled CSS file.